setup.py

```diff
@@ -51,6 +51,6 @@
       ],
       keywords=["optimization", "CMA-ES", "cmaes"],
       packages=["cma", "cma.utilities"],
-      requires=["numpy"],
+      install_requires=["numpy", "scikit-optimize"],
```

Comment:
I found that `setuptools` has an [`extras_require`](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) keyword, which seem we should move the scikit requirement in.